### PR TITLE
Export locations to CSV

### PR DESCRIPTION
**Pull request by dwhenry**
Allow locations to be exported in CSV format.

### CODE REVIEW

File: app/services/locations_csv.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class LocationsCSV < CsvGenerator
+  def initialize(record_or_records)
+    @records = Array(record_or_records).map { |record| LocationWithAddress.new(record) }
+  end
+
+  def attributes
+    %w(uid title address_line_1 address_line_2 address_line_3 town
```
Comment (benlovell) [inline app/services/locations_csv.rb]: Can we format these with each successive attribute on a newline? As per the styling here https://github.com/guidance-guarantee-programme/location-register/pull/31/files#diff-6269b912e824598cdd2e7528fdd8e17aR12

Comment (dwhenry) [inline app/services/locations_csv.rb]: done

File: app/services/locations_csv.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class LocationsCSV < CsvGenerator
```
Comment (benlovell) [inline app/services/locations_csv.rb]: Can you match the casing with the `CSV` suffix to `Csv`?

Comment (dwhenry) [inline app/services/locations_csv.rb]: done

**Comment by benlovell** [comment]
Lovely job. 🚀

**Pull request merged**